client/components/tinymce/plugins/media/drop-zone.jsx

```diff
@@ -6,64 +6,57 @@
 
 import PropTypes from 'prop-types';
 import React from 'react';
-import createReactClass from 'create-react-class';
 import { noop } from 'lodash';
+import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
 import analytics from 'lib/analytics';
-import observe from 'lib/mixins/data-observe';
 import PostActions from 'lib/posts/actions';
 import MediaDropZone from 'my-sites/media-library/drop-zone';
 import MediaActions from 'lib/media/actions';
 import MediaUtils from 'lib/media/utils';
 import MediaLibrarySelectedStore from 'lib/media/library-selected-store';
 import MediaValidationStore from 'lib/media/validation-store';
 import markup from 'post-editor/media-modal/markup';
+import { getSelectedSite } from 'state/ui/selectors';
 
-export default createReactClass( {
-	displayName: 'TinyMCEDropZone',
+class TinyMCEDropZone extends React.Component {
+	static displayName = 'TinyMCEDropZone';
```

Comment:
We could probably drop the `displayName`.

Reply:
`displayName` removed!